synapse/tests/test_model_infotech.py

```diff
@@ -700,3 +702,29 @@ class InfotechModelTest(s_t_utils.SynTest):
             self.eq('::ffff:5.5.5.5', nodes[0].get('dst:ipv6'))
 
             self.eq(0x10000200003, nodes[0].get('version'))
+
+    async def test_it_reveng(self):
+
+        async with self.getTestCore() as core:
+
+            baseFile = hashlib.sha256(b'file').hexdigest()
```

Comment:
`s_common.buid()` will generate you a random 256 byte valu

Reply:
gracias